pkg/services/star/starimpl/star.go

```diff
@@ -4,16 +4,16 @@ import (
 	"context"
 
 	"github.com/grafana/grafana/pkg/infra/db"
+	"github.com/grafana/grafana/pkg/services/featuremgmt"
 	"github.com/grafana/grafana/pkg/services/star"
-	"github.com/grafana/grafana/pkg/setting"
 )
 
 type Service struct {
 	store store
 }
 
-func ProvideService(db db.DB, cfg *setting.Cfg) star.Service {
-	if cfg.IsFeatureToggleEnabled("newDBLibrary") {
+func ProvideService(db db.DB, togggles featuremgmt.FeatureToggles) star.Service {
```

Comment:
if the service does not need the whole config, I think better to just depend on the toggles

Reply:
nevermind... enough tests need to change that this is a much bigger change than it should be!  (so I reverted this part)